test/test.js

```diff
@@ -35,6 +34,14 @@ describe('Leaflet-headless', function () {
 			size.y.should.equal(1024);
 		});
 
+		it('can change size', function () {
+			map.setView(latlng, 10).setSize(800,600);
+			
+			var size = map.getSize();
+			size.x.should.equal(800);
+			size.y.should.equal(600);
+		});
```

Comment:
Maybe another test actually changing the size on a map instance might be a good addition? Another comma here.